src/librustc_unicode/char.rs

```diff
@@ -529,13 +529,19 @@ impl char {
 
     /// Returns true if this `char` is an alphabetic code point, and false if not.
     ///
+    /// A `char` is alphabetic if it belongs to any of [the following
+    /// categories](http://unicode.org/reports/tr44/#Alphabetic):
+    ///
+    ///     Lu | Ll | Lt | Lm | Lo | Nl | Other_Alphabetic
```

Comment:
As per the errors in Travis CI, rustdoc will treat this indented section as code that should be tested. You could do something like this though:

    ```ignore
    Lu | Ll | Lt | Lm | Lo | Nl | Other_Alphabetic
    ```

https://doc.rust-lang.org/book/documentation.html#running-documentation-tests